src/IlluminaSampleSheet/V2/BclConvert/BclSample.php

```diff
@@ -13,7 +13,7 @@ class BclSample
 
     public ?string $index2 = null;
 
-    public ?string $overrideCycles = null;
+    public OverrideCycles $overrideCycles;
```

Comment:
The constructor does not ensure this is set. I think it should be added through there.